install_config/configuring_authentication.adoc

```diff
@@ -756,6 +756,11 @@ xref:requestheader-master-ca-config[master's identity provider configuration].
   --serial='/etc/origin/master/proxyca.serial.txt'
 ----
 
+[NOTE]
+`oadm ca create-signer-cert` generates a certificate that is valid for 5 years.
+This period can be altered with `*--expire-days*` option but by security
+reasons it is strongly recommended to not make it greater than this value.
```

Comment:
This NOTE doesn't enclosed in `====` because otherwise we'll break the example section.